app_legacy/Community/Enums/TicketFilters.php

```diff
@@ -42,18 +42,21 @@ abstract class TicketFilters
 
     public const ResolvedByNonAuthor = 1 << 17;
 
-    public const StateRequest = 1 << 18;
+    public const ResolvedByNonReporter = 1 << 18;
 
-    public const EmulatorOther = 1 << 19;
+    public const StateRequest = 1 << 19;
```

Comment:
It's generally unsafe to renumber enums. While unlikely, any users with bookmarks for a list with the EmulatorOther filter selected would now get StateRequest matches.

Reply:
Ah I recall you mentioning that in the past thanks for reminding me 👍